onnx_caffe2/bin/conversion.py

```diff
@@ -0,0 +1,87 @@
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+from caffe2.proto import caffe2_pb2
+import click
+import numpy as np
+from onnx import checker, onnx_pb2
+
+from onnx_caffe2.backend import Caffe2Backend as c2
+import onnx_caffe2.frontend as c2_onnx
+
+
+@click.command(
+    help='convert caffe2 net to onnx model',
+    context_settings={
+        'help_option_names': ['-h', '--help']
+    }
+)
+@click.option('--caffe2-net', required=True,
```

Comment:
I hate to bikeshed command line syntax, but wouldn't it be more Unixy for the tool to take `--caffe2-net` as a positional argument, for there to be a shortened `-o` form for the output name (defaulting to stdout when this is not provided)?

Reply:
Makes sense, I will change it to positional argument. For output, I need to change '-o'/'--output' to a output directory.